app/jobs/notifications/create_job.rb

```diff
@@ -67,6 +75,21 @@ def message
           user_name: @actor.name,
           community_name: @resource.community.name
         )
+      when :submission_comment_created
+        I18n.t(
+          'jobs.notifications.create.message.submission_comment_created',
```

Comment:
Auto-formatting would cause this to be double-quoted. Editor misconfigured?

Reply:
Yes, I changed the ruby formatter to `"editor.defaultFormatter": "testdouble.vscode-standard-ruby"` in global settings when working on another repo instead of in workspace settings, fixed it now.